src/instance_host/local_host.rs

```diff
@@ -0,0 +1,34 @@
+use crate::instance_host::{Instance, InstanceHost};
+
+use async_trait::async_trait;
+use std::process::Command;
+
+use tracing::info;
+
+pub struct LocalHost {}
+
+impl LocalHost {
+    pub fn new() -> LocalHost {
+        LocalHost {}
+    }
+}
+
+#[async_trait]
+impl InstanceHost for LocalHost {
+    async fn start_instance(
+        &mut self,
+        username: String,
+    ) -> Result<Instance, Box<dyn std::error::Error>> {
+        Command::new("sh")
+                .arg("-c")
+                .arg("echo hello")
+                .output()
```

Comment:
I guess `spawn` instead of `output` is the way to go. We should also store the `Child` objects returned by `spawn`, so later on we can close each process.

Reply:
Yeah, good point